searx/settings.yml

```diff
@@ -65,6 +65,7 @@ ui:
 #result_proxy:
 #    url : http://127.0.0.1:3000/
 #    key : !!binary "your_morty_proxy_key"
+#    button : True  # [True|False] enable the "proxy" button next to each result
```

Comment:
Instead of `button` (where is this button ?), I suggest `result_proxy`  ... but actually it is already the name of the section (line 65).

So may be `html_proxy` or `proxied_link` ?

---

If we break the compatibility with searx:
```
morty:
    url : http://127.0.0.1:3000/
    key : !!binary "your_morty_proxy_key"
    proxify_images: true
    proxify_results: false
```

Reply:
I have a push the commit cf9518b1a0bef31b0184e25d3b8b115e70031ce9

```
#result_proxy:
#    url : http://127.0.0.1:3000/
#    key : !!binary "your_morty_proxy_key"
#    proxify_results: True  # [True|False] enable the "proxy" button next to each result
```

Reply:
> I have a push the commit cf9518b

@dalf thanks! ... I do not have much time ATM .. feel free to squash & rebase ... if you want to merge.